doc/api/next_api_changes/removals/23XXX-GL.rst

```diff
@@ -25,3 +25,8 @@ Pass in the explicit keyword arguments desired when creating the axes.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Use ``docutils.parsers.rst.directives.images.Image.align`` instead.
+
+``imread()`` no longer accepts URLs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These should be opened and passed directly to Pillow now.
```

Comment:
Please take over the example.

Reply:
:+1: I updated all of the change log suggestions. I went back and took the wording from the 3.4 deprecations (not sure why I didn't do that in the first place :smile:)